proj.sh

```diff
@@ -41,7 +41,7 @@ function logs {
 
 function deploy-stack {
   generate-data
-  docker-compose -f docker-compose.yml -f docker-compose-prod.yml --context $1 up -d --build
+  docker --context $1 compose -f docker-compose.yml -f docker-compose-local.yml logs -f nginx
```

Comment:
The change of `docker-compose-prod.yml` to `docker-compose-local.yml`, is that just a temporary fix? How is the SSL certificate set up in this case?

Reply:
Oh I didn't realize that haha. It wasn't intentional and we should probably not merge this in yet..

Reply:
also we  might not need the `logs -f nginx` at the end either